### PR TITLE
Add failed test list to cli report

### PR DESCRIPTION
**Pull request by Vacxe**
+ replace deprecated `appendln`

Final report
```
Device pool omni:
	22 passed, 2 failed, 0 ignored tests
	Failed tests:
		com.agoda.sample.DrawableListTest#matchDrawablesInListView
		com.agoda.sample.AutoCompleteTest#testContentItemsListView
	Flakiness overhead: 68065ms
	Raw: 22 passed, 4 failed, 0 ignored, 0 incomplete tests
	Failed tests:
		com.agoda.sample.DrawableListTest#matchDrawablesInListView failed 2 times
		com.agoda.sample.AutoCompleteTest#testContentItemsListView failed 2 times
Total time: 0H 4m 0s

```

### CODE REVIEW

File: core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt [hunk 1]
```diff
@@ -10,21 +10,49 @@ class StdoutReporter(private val timer: Timer) : Reporter {
...
+            cliReportBuilder.appendLine("\tRaw: ${poolSummary.rawPassed.size} passed, ${poolSummary.rawFailed.size} failed, ${poolSummary.rawIgnored.size} ignored, ${poolSummary.rawIncomplete.size} incomplete tests")
+
+            if(poolSummary.rawFailed.isNotEmpty()){
+                cliReportBuilder.appendLine("\tFailed tests:")
+                poolSummary.rawFailed
+                    .groupBy { it }
+                    .toSortedMap()
+                    .mapValues { it.value.size }
+                    .forEach { (testName, count) ->
+                        cliReportBuilder.appendLine("\t\t$testName failed $count times")
```
Comment (Malinskiy) [inline core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt]: Since we don't have proper plular support, maybe reword to `time(s)`?

File: core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt [hunk 2]
```diff
@@ -10,21 +10,49 @@ class StdoutReporter(private val timer: Timer) : Reporter {
...
+            }
+
+            if(poolSummary.rawIncomplete.isNotEmpty()){
+                cliReportBuilder.appendLine("\tIncomplete tests:")
+                    poolSummary.rawIncomplete
+                        .groupBy { it }
+                        .toSortedMap()
+                        .mapValues { it.value.size }
+                        .forEach { (testName, count) ->
+                            cliReportBuilder.appendLine("\t\t$testName incomplete $count times")
```
Comment (Malinskiy) [inline core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt]: same problem with plurals

File: core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt (hunk 1 above)
Comment (Vacxe) [reply core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt]: WDYT about linguistic library implementation such as https://github.com/atteo/evo-inflector?

Comment (Malinskiy) [reply core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt]: Can we just use `(s)`? we're not a GUI app, please just change the template to use `(s)`

Comment (Vacxe) [reply core/src/main/kotlin/com/malinskiy/marathon/report/stdout/StdoutReporter.kt]: No problem mate
